src/zjs_event.h

```diff
@@ -1,3 +1,5 @@
+// Copyright (c) 2016, Intel Corporation.
+
```

Comment:
missing copyrights could be separated as well and i could check to make sure i got all of them i guess